plugins/es-repository-azure/pom.xml

```diff
@@ -51,6 +51,12 @@
       <version>${versions.jackson}</version>
     </dependency>
 
+    <dependency>
+      <groupId>com.google.guava</groupId>
```

Comment:
Why does the es-repository-azure now have a new dependency?

Reply:
This is to pinpoint the guava version, otherwise, we have two different versions in the classpath at runtime.